### PR TITLE
Mobile species filter

### PR DESCRIPTION
**Pull request by mrtmrt233**
addresses #296 

Mobile view now shows species search and locate button, hides address bar

### CODE REVIEW

**Comment by emillipede** [comment]
thank you @mrtmrt233 

when I test it, the species buttons are kind of floating in the middle of the toolbar, and the select2 bar sometimes overlaps a little bit with the buttons (see screenshots). the toolbar is also quite tall, but that's a separate issue (#245)

is it difficult move the leaflet location button the bottom of the map (with the zoom in/out buttons)?

before (testing as iphoneX)
Screen Shot 2020-06-11 at 11 38 35 AM

after (testing as iphoneX)
Screen Shot 2020-06-11 at 11 38 10 AM
Screen Shot 2020-06-11 at 11 44 22 AM

**Comment by mrtmrt233** [comment]
fixed (I think)

**Comment by emillipede** [comment]
thank you @mrtmrt233 

will you please adjust the margins a little bit, so that the borders on each side of the toolbar look even? 

do you know why the placeholder text inside the select2 bar is cut off (even though there's space left inside the bar, the E at the end of "name" isn't visible - see photo)? the same issue come up with the address search bar, but it wasn't resolved

before (testing as iphoneX):
Screen Shot 2020-06-11 at 1 41 53 PM

after (testing as iphoneX):
Screen Shot 2020-06-11 at 1 40 08 PM

Screen Shot 2020-06-11 at 1 41 39 PM

File: public/js/Map.js [hunk 1]
```diff
@@ -63,6 +63,7 @@ var app = this.app || {};
         inView: 'stop',
         inViewNotFollowing: 'stop'
       },
+      position:'bottomright',
```
Comment (emillipede) [inline public/js/Map.js:66]: will you please update the position of the zoom (line 80) to match this position

> `L.control.zoom({position: 'bottomright'}).addTo(this.leafletMap);`

Comment (mrtmrt233) [reply public/js/Map.js:66]: done

**Pull request merged**